README.md

```diff
@@ -52,111 +47,98 @@ geli help
-to ensure the Master Key for is unique.  Here you can set up the
-cryptographic algorithm to use, Data Key length, etc. The last sector
-of the providers is used to store metadata. that the metadata can be
-recovered with the restore subcommand using the backupfile and backup
-action described below.
+Initialize providers which need to be encrypted.
+Multiple providers are not supported.
+A unique salt will be randomly generated for provider to ensure the Master Key for is unique.
+Here you can set up the cryptographic algorithm to use, Data Key length, etc.
+The last sector of the providers is used to store metadata.
+that the metadata can be recovered with the restore subcommand using the backupfile and backup action described below.
 
 Additional options include:
 
 - `-b`
-Try to decrypt this partition during FreeBSD boot, before the root
-partition is mounted. This makes it possible to use an encrypted root
-partition for only the FreeBSD
+Try to decrypt this partition during FreeBSD boot, before the root partition is mounted.
+This makes it possible to use an encrypted root partition for only the FreeBSD
 
 - `-B backupfile`
-File name to use for metadata backup. To inhibit backups, you can avoid
-this option at command line.
+File name to use for metadata backup.
+To inhibit backups, you can avoid this option at command line.
 
 - `-e ealgo`
-Encryption algorithm to use. Currently supported algorithm is:
-`AES-XTS`. The default and recommended algorithm is `AES-XTS`.
+Encryption algorithm to use.
+Currently supported algorithm is: `AES-XTS`.
+The default and recommended algorithm is `AES-XTS`.
 
 - `-g`
-Enable booting from this encrypted root filesystem. Only applicable when
-booting the encrypted device using FreeBSD. The FreeBSD boot loader
-prompts for the passphrase and loads loader(8) from the encrypted
-partition.
+Enable booting from this encrypted root filesystem.
+Only applicable when booting the encrypted device using FreeBSD.
+The FreeBSD boot loader prompts for the passphrase and loads loader(8) from the encrypted partition.
 
 - `-i iterations`
-Number of iterations to use with PKCS#5v2 when processing User Key
-passphrase component. This option is mandatory.
+Number of iterations to use with PKCS#5v2 when processing User Key passphrase component.
+This option is mandatory.
 
 - `-J newpassfile`
-Specifies a file which contains the passphrase component of the User Key
-(or part of it).  If newpassfile is given as -, standard input will
-be used. Only the first line (excluding new-line character) is taken
-from the given file. otherwise the environment variable of `passphrase`
-will be used.
+Specifies a file which contains the passphrase component of the User Key (or part of it).
+If newpassfile is given as -, standard input will be used.
+Only the first line (excluding new-line character) is taken from the given file.
+otherwise the environment variable of `passphrase` will be used.
```

Comment:
```suggestion
Only the first line (excluding new-line character) is taken from the given file, otherwise the environment variable of `passphrase` will be used.
```